streamer/src/nonblocking/quic.rs

```diff
@@ -185,8 +224,37 @@ async fn run_server(
         stats.clone(),
         coalesce,
     ));
+
+    let mut accepts = incoming
```

Comment:
Can you please put some comments here why we are doing this select! it is not immediately obvious.